apps/gidd/views.py

```diff
@@ -119,9 +119,8 @@ def export(self, request):
         Export disaster
         """
         qs = self.filter_queryset(self.get_queryset())
-        wb = Workbook()
-        ws = wb.active
-        ws.title = "1_Disaster_Displacement_data"
+        wb = Workbook(write_only=True)
```

Comment:
```suggestion
        wb = Workbook(optimized_write=True)
```
This would also be efficient if not much formating is needed.

Reply:
Looks like internally they are same
https://github.com/theorchard/openpyxl/blob/b06d737937d6c04927d8cfe9758a78d94c023872/openpyxl/workbook/workbook.py#L50C29-L50C39